### PR TITLE
DIGI_2_ListAllAvailableStores

### PR DESCRIPTION
**Pull request by rajendarreddy1234**
2.Added store controller

### CODE REVIEW

File: DigiOrderApi/DigiOrderApi/Controllers/StoreController.cs [hunk 1]
```diff
@@ -9,5 +9,33 @@ namespace DigiOrderApi.Controllers
...
+        {
+            try
+            {
+                storeDataModel = new StoreDataModel();
+                dtRecords = storeDataModel.getAllAvailableStores();
+                dtRecords.TableName = "GETAVAILABLESTORES";
+            }
+            catch (Exception ex)
+            {
+                return Request.CreateErrorResponse(HttpStatusCode.NotFound, "No Record Found");
```
Comment (anebhargavkrishna) [inline DigiOrderApi/DigiOrderApi/Controllers/StoreController.cs:36]: Statuscode.internal server message as "Unable to process request"

File: DigiOrderApi/DigiOrderApiDataAccess/StoreDataModel.cs [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+namespace DigiOrderApiDataAccess
+{
+    public class StoreDataModel
+        {
+            //Author:Rajendar Redddy konuganti
+            //6-24-2020
+            //Description: To get all available stores from database
+            //Reviewer:
+            //Review Version:1.0
```
Comment (anebhargavkrishna) [inline DigiOrderApi/DigiOrderApiDataAccess/StoreDataModel.cs:12]: Remove comments

File: DigiOrderApi/DigiOrderApiDataAccess/StoreDataModel.cs [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+                    dtResult = DB_UTILITY.RunSP("USP_GETALLSTORES", null);
+                }
+                catch (Exception ex)
+                {
+                    throw ex;
+                }
+                return dtResult;
+            }
+        }
+
```
Comment (anebhargavkrishna) [inline DigiOrderApi/DigiOrderApiDataAccess/StoreDataModel.cs:28]: Remove space in line 28

File: DigiOrderApi/DigiOrderApi/Controllers/StoreController.cs (hunk 1 above)
Comment (rajendarreddy1234) [reply DigiOrderApi/DigiOrderApi/Controllers/StoreController.cs]: Resolved

File: DigiOrderApi/DigiOrderApiDataAccess/StoreDataModel.cs (hunk 2 above)
Comment (rajendarreddy1234) [reply DigiOrderApi/DigiOrderApiDataAccess/StoreDataModel.cs]: Removed

File: DigiOrderApi/DigiOrderApiDataAccess/StoreDataModel.cs (hunk 3 above)
Comment (rajendarreddy1234) [reply DigiOrderApi/DigiOrderApiDataAccess/StoreDataModel.cs]: Resolved

File: DigiOrderApi/DigiOrderApi/Controllers/StoreController.cs [hunk 4]
```diff
@@ -1,13 +1,43 @@
...
+        [Authorize(Roles = "ENTERPRISE")]
+        [Route("api/Store/All")]
+        [HttpGet]
+        public HttpResponseMessage Get()
+        {
+            try
+            {
+                storeDataModel = new StoreDataModel();
+                dtRecords = storeDataModel.getAllAvailableStores();
+                dtRecords.TableName = "GETAVAILABLESTORES";
```
Comment (RAMUNIMMALA) [inline DigiOrderApi/DigiOrderApi/Controllers/StoreController.cs:34]: Could be good if we have proper table name like STORES as it is Stores list

Comment (rajendarreddy1234) [reply DigiOrderApi/DigiOrderApi/Controllers/StoreController.cs]: Resolved Sir.

**Pull request merged**